react/MiniCart.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { graphql } from 'react-apollo'
+import orderFormQuery from './graphql/orderFormQuery.gql'
+import MiniCartItem from './MiniCartItem'
+import Button from '@vtex/styleguide/lib/Button'
+import { Price } from '@vtex/product-details'
+import Spinner from '@vtex/styleguide/lib/Spinner'
+
+/**
+ * Minicart component
+ */
+class MiniCart extends Component {
+  handleClickButton = () => location.assign('/checkout/#/cart')
+
+  renderWithoutProducts = (label) => {
+    return (
```

Comment:
You return directly in `handleClickButton`... Why not here?

Reply:
Done!